From: Stephan Linz <linz@li-pro.net>
Date: Mon, 1 Sep 2014 18:27:51 +0200
Subject: simz80: avoid warnings unused variable or used uninitialized

Signed-off-by: Stephan Linz <linz@li-pro.net>
---
 simz80.c  | 6 +++---
 simz80.pl | 6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/simz80.c b/simz80.c
index 3506317..0e42350 100644
--- a/simz80.c
+++ b/simz80.c
@@ -106,7 +106,7 @@ static void
 cb_prefix(FASTREG adr)
 {
     DECLARE_STATE();
-    FASTWORK temp, acu, op, cbits;
+    FASTWORK temp = 0, acu = 0, op, cbits;
 
 		switch ((op = GetBYTE(PC)) & 7) {
 		case 0: ++PC; acu = hreg(BC); break;
@@ -673,8 +673,8 @@ simz80(FASTREG PC)
     FASTREG DE = regs[regs_sel].de;
     FASTREG HL = regs[regs_sel].hl;
     FASTREG SP = sp;
-    FASTWORK temp, acu, sum, cbits;
-    FASTWORK op, adr;
+    FASTWORK temp = 0, acu = 0, sum, cbits;
+    FASTWORK op/*, adr*/;
 
 #ifdef DEBUG
     while (!stopsim) {
diff --git a/simz80.pl b/simz80.pl
index a31734a..abde648 100644
--- a/simz80.pl
+++ b/simz80.pl
@@ -1411,7 +1411,7 @@ static void
 cb_prefix(FASTREG adr)
 {
     DECLARE_STATE();
-    FASTWORK temp, acu, op, cbits;
+    FASTWORK temp = 0, acu = 0, op, cbits;
 
 EOT
 	&cb_switch;
@@ -1489,8 +1489,8 @@ EOT
     FASTREG IY = iy;
 EOT
     }
-    print "    FASTWORK temp, acu, sum, cbits;\n";
-    print "    FASTWORK op, adr;\n" if ($cb_inline + $dfd_inline +
+    print "    FASTWORK temp = 0, acu = 0, sum, cbits;\n";
+    print "    FASTWORK op/*, adr*/;\n" if ($cb_inline + $dfd_inline +
 				  $ed_inline != 0);
 
     if ($optab) {
