From: Stephan Linz <linz@li-pro.net>
Date: Mon, 1 Sep 2014 19:27:10 +0200
Subject: set _FORTIFY_SOURCE=0 to disable compile-time checks

Signed-off-by: Stephan Linz <linz@li-pro.net>
---
 Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ab88079..7f79000 100644
--- a/Makefile
+++ b/Makefile
@@ -24,7 +24,9 @@ OPTIMIZE       = -g -O2
 # -DMEMSIZE <val> sets size of memory in KBytes (default 64)
 # solaris2 needs -D__EXTENSIONS__
 # linux needs -D_GNU_SOURCE
-OPTIONS	      = -DBIOS -D_GNU_SOURCE
+# Ubuntu 8.10 and later needs -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0
+OPTIONS	      = -DBIOS -D_GNU_SOURCE \
+		-U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0
 
 # Link with CP/M BIOS support,
 YAZE_OBJS     = yaze.o simz80.o bios.o monitor.o
