From: Stephan Linz <linz@li-pro.net>
Date: Mon, 1 Sep 2014 19:28:48 +0200
Subject: set MEMSIZE=64 explicitly

Signed-off-by: Stephan Linz <linz@li-pro.net>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 7f79000..ff045c7 100644
--- a/Makefile
+++ b/Makefile
@@ -25,7 +25,7 @@ OPTIMIZE       = -g -O2
 # solaris2 needs -D__EXTENSIONS__
 # linux needs -D_GNU_SOURCE
 # Ubuntu 8.10 and later needs -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0
-OPTIONS	      = -DBIOS -D_GNU_SOURCE \
+OPTIONS	      = -DBIOS -DMEMSIZE=64 -D_GNU_SOURCE \
 		-U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0
 
 # Link with CP/M BIOS support,
