From: Stephan Linz <linz@li-pro.net>
Date: Mon, 1 Sep 2014 19:46:07 +0200
Subject: set LIBDIR default to /usr/local/share/yaze

Signed-off-by: Stephan Linz <linz@li-pro.net>
---
 Makefile | 2 +-
 yaze.c   | 2 +-
 yaze.doc | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 3064a80..446802c 100644
--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,7 @@ CC            =	gcc
 #where you want the binaries and manual page
 BINDIR	      = /usr/local/bin
 MANDIR	      = /usr/local/man/man1
-LIBDIR	      = /usr/local/lib
+LIBDIR	      = /usr/local/share/yaze
 
 # full speed or debugging to taste
 #OPTIMIZE      = -O2
diff --git a/yaze.c b/yaze.c
index 6dc9f04..b3fedb1 100644
--- a/yaze.c
+++ b/yaze.c
@@ -52,7 +52,7 @@ BYTE *pagetable[16];		/* MMU page table */
 #endif
 
 #ifndef LIBDIR
-#define LIBDIR "/usr/local/lib/"
+#define LIBDIR "/usr/local/share/yaze/"
 #endif
 
 static char *bootfile = "yaze.boot";
diff --git a/yaze.doc b/yaze.doc
index a5b008f..c6b8fc6 100644
--- a/yaze.doc
+++ b/yaze.doc
@@ -233,7 +233,7 @@ extensions.
 
 'make install' puts the binaries in BINDIR (default: /usr/local/bin),
 the manual pages in MANDIR (default: /usr/local/man/man1) and the
-bootstrap file in LIBDIR (default: /usr/local/lib).
+bootstrap file in LIBDIR (default: /usr/local/share/yaze).
 
 
 3.  Command-line options
