From 0e655c7f36897a8bb3c8779885b98eb27e26cce0 Mon Sep 17 00:00:00 2001
From: Stephan Linz <linz@li-pro.net>
Date: Mon, 1 Sep 2014 19:58:04 +0200
Subject: [PATCH] use GNU readline library

Signed-off-by: Stephan Linz <linz@li-pro.net>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index a4fbdaf..61de6e3 100644
--- a/Makefile
+++ b/Makefile
@@ -25,7 +25,7 @@ OPTIMIZE       = -g -O2
 # solaris2 needs -D__EXTENSIONS__
 # linux needs -D_GNU_SOURCE
 # Ubuntu 8.10 and later needs -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0
-OPTIONS	      = -DBIOS -DMEMSIZE=64 -D_GNU_SOURCE \
+OPTIONS	      = -DBIOS -DMEMSIZE=64 -DUSE_GNU_READLINE -D_GNU_SOURCE \
 		-U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0
 
 # Link with CP/M BIOS support,
@@ -35,7 +35,7 @@ YAZE_OBJS     = yaze.o simz80.o bios.o monitor.o
 
 # -lreadline -lcurses -liberty if you defined USE_GNU_READLINE
 #  (you don't need -liberty on linux or BSD systems)
-LIBS	      =
+LIBS	      = -lreadline -lncurses
 
 # a bsd-like install program (/usr/ucb/install on Solaris2)
 INSTALL	      = install
